appveyor.yml

```diff
@@ -1,56 +1,17 @@
-version: 1.0.{build}
```

Comment:
IMO it's not needed (I believe it's only used in AppVeyor's web interface to refer to the builds), and it's wrong anyway.